### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by lovromazgon**
This PR reformats the readme and adds a simple example for a quickly getting started with Conduit. It removes the "testing" chapter (if we need this we can create it in another file, it's not needed in the readme IMO), it also simplifies the known limitations, since most limitations were addressed in the last milestone but I think we still need the disclaimer.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,50 +14,87 @@ _Data Integration for Production Data Stores. :dizzy:_
...
 - [Connectors](#connectors)
 - [Processors](#processors)
-- [Testing](#testing)
 - [API](#api)
 - [UI](#ui)
 - [Documentation](#documentation)
 - [Known limitations](#known-limitations)
 - [Contributing](#contributing)
 
+## Quick start
```
Comment (maha-hajja) [inline README.md:45]: love this chapter!

**Review by hariso — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -13,50 +14,87 @@ _Data Integration for Production Data Stores. :dizzy:_
...
+
+1. Download and extract
+   the [latest release](https://github.com/conduitio/conduit/releases/latest).
+2. Download
+   the [example pipeline](https://github.com/ConduitIO/conduit/blob/main/examples/processors/extract-field-transform.sh)
```
[the remark this answers is not in the document]
Comment (maha-hajja) [reply README.md]: was fixing this and got a conflict.. now I know why 👀

[the remark this answers is not in the document]
Comment (maha-hajja) [reply README.md]: was fixing this and got a conflict.. now I know why 👀

File: README.md (hunk 1 above)
Comment (hariso) [reply README.md:45]: Agree 100%!!

**Pull request merged**